### ISSUE TITLE
Be able to coexist with CDJ-3000s on channel 5 and 6

### ISSUE THREAD

**Issue by brunchboy** [Deep-Symmetry/beat-link#63]
The problem is worse than I realized: if BLT (or, presumably, any other player that is identified as “too old” by the CDJ-3000s) appears on the network, they display an error dialog, which I knew, but _they also kick themselves off the network_. Thankfully, @nudge has identified the byte that needs to change to prevent that, and posted about it in the [Zulip stream](https://deep-symmetry.zulipchat.com/#narrow/stream/275322-beat-link-trigger/topic/Error.20investigation/near/400503698).

**Comment by brunchboy** [comment]
Sadly, that change alone does not seem to be enough to solve the problem. 😞 Looking into other packet differences now.

**Comment by brunchboy** [comment]
OK, I found several other differences in the keep-alive packet, but not even emulating all of those was enough to fix the warning. I did get to a point where the warning would appear but the CDJ-3000 would stay on the network, but I could not eliminate the warning.

Finally I tried booting the CDJ-3000 with Beat Link already online and transmitting status packets (that required having another CDJ-3000 on the network, so Beat Link would not take itself offline). And, huzzah! No warning! So the warning is now being caused by some aspect of our startup / device number claim packets. Time to investigate those.
